### PR TITLE
Add mel_weight_matrix to onnx frontend

### PR DESCRIPTION
**Pull request by mohame54**
close https://github.com/unifyai/ivy/issues/21162

### CODE REVIEW

**Comment by mohame54** [comment]
Add all the missing ONNX functions to Ivy's experimental  @Aarsh2001

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Comment by mohame54** [comment]
@MuhammadSaeedBatikh  I am ready for the review

File: ivy/data_classes/array/experimental/creation.py [hunk 1]
```diff
@@ -123,3 +123,56 @@ def unsorted_segment_sum(
...
+        sample_rate: Union[int, ivy.Array],
+        lower_edge_hertz: Optional[Union[float, ivy.Array]] = 0.0,
+        upper_edge_hertz: Optional[Union[float, ivy.Array]] = 3000.0,
+    ):
+        """
+        Generate a MelWeightMatrix that can be used to re-weight a Tensor containing a
+        linearly sampled frequency spectra (from DFT or STFT) into num_mel_bins
+        frequency information based on the [lower_edge_hertz, upper_edge_hertz]
+
+        range on the mel scale. This function defines the mel scale in terms of
```
[the remark this answers is not in the document]
Comment (mohame54) [reply ivy/data_classes/array/experimental/creation.py:140]: so it should be a static method right ?

**Pull request closed without merging**